### DISCUSSION TITLE
Has anyone gotten the file-from-disk example working?

### DISCUSSION THREAD

**Discussion by aivajoe** [pion/webrtc#2449] [Q&A]
First of all, I love Pion! Thank you for your continued work on this great project.

I'm wondering if anyone has gotten this example to work in practice: https://github.com/pion/example-webrtc-applications/tree/master/play-from-disk-h264

It runs, and it sends the opus data just fine (ogg with 20ms page size), but the other participant in the session doesn't play back the audio. I've tried it with 3 different browsers (Firefox, Chrome, Safari).

Chrome shows a high "packetsDiscarded" count, which suggests some kind of timing issue? Not really sure.

Any help would be appreciated. Thank you.

**Comment by Sean-Der** [comment]
Hi @aivajoe 

does this work for you now? Sorry I wasn’t more help. If you still need help I can try this out tonight. 

Thanks for using Pion :)
